src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,109 @@
+import React from 'react';
+
+import { getNumbers } from '../../utils';
+import classNames from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+  const pages = getNumbers(1, totalPages);
+
+  const isOnFirstPage = currentPage === 1;
+  const isOnLastPage = currentPage === totalPages;
+
+  const itemsOnPage = getNumbers(
+    (currentPage - 1) * perPage + 1,
+    Math.min(currentPage * perPage, total),
```

Comment:
The variable 'itemsOnPage' is not used in the component rendering. If it's not needed, consider removing it to avoid unnecessary computations and improve code clarity.